lib/addons/aws-for-fluent-bit/index.ts

```diff
@@ -0,0 +1,73 @@
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+import { Construct } from "@aws-cdk/core";
+
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo } from "../../spi/types";
+import { createNamespace } from "../../utils/namespace-utils";
+
+/**
+ * Configuration options for the FluentBit add-on.
```

Comment:
There is a ton of options that this addon takes, e.g. cloudwatch forwarding, kinesis forwarding, es, etc. 
It would help to add a link to the values supported by the add-on. It can be done here, or by adding values field that matches the definition of the super interface and adding docs section to it.

Reply:
Done. Added to doc block for the add-on